DuckDuckGo/PrivacyProtectionFooterController.swift

```diff
@@ -30,24 +30,33 @@ class PrivacyProtectionFooterController: UIViewController {
     var contentBlocker: ContentBlockerConfigurationStore = ContentBlockerConfigurationUserDefaults()
 
     override func viewDidLoad() {
+        super.viewDidLoad()
         leaderboard.didLoad()
-        update()
     }
 
-    override func viewDidAppear(_ animated: Bool) {
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
         update()
     }
-
+    
     @IBAction func toggleProtection() {
         let contentBlockingOn = privacyProtectionSwitch.isOn
         self.contentBlocker.enabled = contentBlockingOn
         update()
     }
 
+    override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
```

Comment:
ViewWillAppear is getting called on rotation so I think you can delete this method.

Reply:
Thanks @subsymbolic 

It is needed because if you reset the tracker stats and then go back to the dashboard it needs to do another update to go from the score view to the gathering view. :)

Reply:
In case it was unclear, I'm referring to the `viewWillTransition` method not being needed.

Reply:
Actually, scratch that, you're right.